vulnerabilities/importers/rust.py

```diff
@@ -66,15 +67,13 @@ def updated_advisories(self) -> Set[Advisory]:
         return self._load_advisories(self._updated_files)
 
     def _load_advisories(self, files) -> Set[Advisory]:
-        files = [f for f in files if not f.endswith("-0000.toml")]  # skip temporary files
+        files = [f for f in files if not f.endswith("-0000.md")]  # skip temporary files
```

Comment:
are you sure it will always end with "0000.md" ?

Reply:
It will always be named `RUSTSEC-0000-0000.md`

Reply:
```suggestion
                # per @tarcieri It will always be named RUSTSEC-0000-0000.md
                # https://github.com/nexB/vulnerablecode/pull/281/files#r528899864
                files = [f for f in files if not f.endswith("-0000.md")]  # skip temporary files
```